src/pybind/mgr/nfs/export.py

```diff
@@ -304,6 +304,17 @@ def _create_export_user(self, export: Export) -> None:
             rgwfsal.secret_access_key = j['keys'][0]['secret_key']
             log.debug("Successfully fetched user %s for RGW path %s", rgwfsal.user_id, export.path)
 
+    def _ensure_cephfs_export_user(self, export: Export) -> None:
+        assert isinstance(export.fsal, CephFSFSAL)
+        fsal = cast(CephFSFSAL, export.fsal)
+        assert fsal.fs_name
+        assert fsal.cmount_path
+        fsal.user_id = f"nfs.{export.cluster_id}.{export.fsal.fs_name}"
```

Comment:
shouldn't this be checked if a user already exists since we're aiming to share these resources among exports?

Reply:
This is outdated code, please check the latest commit